recipes/libtiff/all/conanfile.py

```diff
@@ -9,27 +9,31 @@ class LibtiffConan(ConanFile):
     license = "MIT"
     homepage = "http://www.simplesystems.org/libtiff"
     topics = ("tiff", "image", "bigtiff", "tagged-image-file-format")
-    exports_sources = ["CMakeLists.txt"]
+    exports_sources = "CMakeLists.txt", "patches/**"
     generators = "cmake", "cmake_find_package"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False],
-               "fPIC": [True, False],
-               "lzma": [True, False],
-               "jpeg": [True, False],
-               "zlib": [True, False],
-               "zstd": [True, False],
-               "jbig": [True, False],
-               "webp": [True, False],
-               "cxx":  [True, False]}
-    default_options = {"shared": False,
-                       "fPIC": True,
-                       "lzma": True,
-                       "jpeg": True,
-                       "zlib": True,
-                       "zstd": True,
-                       "jbig": True,
-                       "webp": True,
-                       "cxx":  True}
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "lzma": [True, False],
+        "jpeg": [False, "libjpeg-turbo", "libjpeg"],
+        "zlib": [True, False],
+        "zstd": [True, False],
+        "jbig": [True, False],
+        "webp": [True, False],
+        "cxx":  [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "lzma": True,
+        "jpeg": "libjpeg-turbo",
```

Comment:
Others recipes allowing to chose between libjpeg and libjpeg-turbo have libjpeg as default option. Isn't it dangerous to make libjpeg-turbo default dependency in this recipe?

Reply:
libjpeg and libjpeg-turbo cannot be used together because they define the same symbols.
An error occurs when linking with both static libraries.

Reply:
Sure, so we should be consistent across all CCI recipes. And currently, they all depends on `libjpeg` as default value.

Reply:
Alright, I was packaging something that wanted libjpeg-turbo as default and has a lot of dependencies (with libjpeg).
So I'll change the default on that recipe, and reverse the default of my latest prs to libjpeg.
Thanks for the feedback.

Reply:
Hoping for some sort of virtual package for libs with same API, so we can avoid these inconsistent options across recipes (for example `mozjpeg` is a valid replacement of `libjpeg` API, but also of `libjpeg-turbo` API).

Reply:
That would be a nice idea and needs some thought.
Have you opened an issue somewhere?

Reply:
No. I remember some discussions around in conan or slack, but I don't know if there is a dedicated issue.